### PR TITLE
docs: workflow examples

### PR DESCRIPTION
**Pull request by nissimsan**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c-ccg/traceability-interop/pull/106.html" title="Last updated on Mar 1, 2022, 7:01 PM UTC (f3e2764)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c-ccg/traceability-interop/106/9482b08...f3e2764.html" title="Last updated on Mar 1, 2022, 7:01 PM UTC (f3e2764)">Diff</a>

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -257,6 +257,26 @@ <h3>CCG Standards Stack</h3>
           </li>
         </ul>
       </section>
+
+      <section class="informative">
+        <h3>Workflows</h3>
+        <img src="workflow-def-ex.png" alt="workflow-def-ex" width="75%">
+        <p>
+          <em>Workflows</em> enable presenters to correlate individual presentations. A workflow <em>definition</em> can be used
+          to indicate that presentations relates to the same type of workflow. In the above example two separate 7501 Entry
```
Comment (TallTed) [inline docs/index.html:266]:
```suggestion
          to indicate that presentations relate to the same type of workflow. In the above example, two separate 7501 Entry
```

File: docs/index.html [hunk 2]
```diff
@@ -257,6 +257,26 @@ <h3>CCG Standards Stack</h3>
...
+          Summaries are submitted to CBP at different points in time, and related to different imports. By specifying the same
+          workflow definition it clear to CBP that both relate to the same types of import workflows.
```
Comment (TallTed) [inline docs/index.html:268]:
```suggestion
          workflow definition, it is made clear to CBP that both relate to the same types of import workflows.
```

File: docs/index.html [hunk 3]
```diff
@@ -257,6 +257,26 @@ <h3>CCG Standards Stack</h3>
...
+          </p>
+        <img src="workflow-inst-ex.png" alt="workflow-inst-ex" width="75%">
+        <p>
+          Workflows <em>instances</em> indicates that individual presentations are part of the very same workflow. The majority
```
Comment (TallTed) [inline docs/index.html:272]:
```suggestion
          Workflow <em>instances</em> indicate that separate individual presentations are part of the very same workflow. The majority
```

File: docs/index.html [hunk 4]
```diff
@@ -257,6 +257,26 @@ <h3>CCG Standards Stack</h3>
...
+          of data elements of a "10+2" Importer Security Filing must be presented to CBP no later than 24h before vessel
```
Comment (TallTed) [inline docs/index.html:273]:
```suggestion
          of data elements in a "10+2" Importer Security Filing must be presented to CBP no later than 24 hours before vessel
```

File: docs/index.html [hunk 5]
```diff
@@ -257,6 +257,26 @@ <h3>CCG Standards Stack</h3>
...
+          loading, the remaining data elements no later than 24h before arrival. Specifying the same workflow instance indicates
```
Comment (TallTed) [inline docs/index.html:274]:
```suggestion
          loading, and the remaining data elements no later than 24 hours before arrival. Specifying the same workflow instance indicates 
```

**Review by TallTed — CHANGES_REQUESTED** [review]

**Comment by mkhraisha** [comment]
I was under the impression that the resolution to #85 was to move vocabulary similar to this to trace vocab?

**Review by mkhraisha — APPROVED** [review]
Disregard my previous comment, this was discussed on the call, happy as is.

**Review by OR13 — APPROVED** [review]

**Review by mprorock — APPROVED** [review]

**Pull request merged**